applylib/applyset/applyset.go

```diff
@@ -93,7 +94,14 @@ type Options struct {
 func New(options Options) (*ApplySet, error) {
 	parent := options.Parent
 	parentRef := &kubectlapply.ApplySetParentRef{Name: parent.Name(), Namespace: parent.Namespace(), RESTMapping: parent.RESTMapping()}
-	kapplyset := kubectlapply.NewApplySet(parentRef, kubectlapply.ApplySetTooling{Name: options.Tooling}, options.RESTMapper)
+
+	toolName, toolVersion, _ := strings.Cut(options.Tooling, "/")
```

Comment:
Could you add a comment showing why the slash splitting is needed? This can help further maintainers be aware of the correlated changes in the kubectlapplyset e.g. pointing to the kubectlapplyset special treatment.

Reply:
Added comment.